sdk/ml/azure-ai-ml/azure/ai/ml/exceptions.py

```diff
@@ -247,17 +243,15 @@ def __init__(
         message: str,
         no_personal_data_message: str,
         *args,
-        target: ErrorTarget = ErrorTarget.UNKNOWN,
-        error_category: ErrorCategory = ErrorCategory.UNKNOWN,
+        target: Union[str, ErrorTarget] = ErrorTarget.UNKNOWN,
+        error_category: Union[str, ErrorTarget] = ErrorCategory.UNKNOWN,
         **kwargs,
     ):
         super(JobException, self).__init__(
             message=message,
+            no_personal_data_message=no_personal_data_message,
             target=target,
             error_category=error_category,
-            no_personal_data_message=no_personal_data_message,
```

Comment:
why args and kwargs? same comment for all classes in this file.

Reply:
same above

Reply:
same as above, let's ignore mypy [misc] here

Reply:
fixed